src/tools/pontoon/ui.md

```diff
@@ -4,115 +4,125 @@
+* A colored square that identifies the string status (i.e. *Missing*, *Translated*, etc.).
+* The source string.
+* The approved translation or the most recent suggestion if available.
 
-### Sidebar
-
-The sidebar displays the list of strings in the current project resource. Each string is displayed with the string status (i.e. *Missing*, *Translated*, etc.) identified by a colored square, the source string, and the approved translation or the most recent suggestion if available.
-
-![Sidebar](../../assets/images/pontoon/ui/sidebar.png)
+![Sidebar](../../assets/images/pontoon/ui/sidebar.png "Screenshot of the sidebar, with a list of strings showing the different string statuses")
 
 Color legend:
 * **<span style="color: #7bc876;">green</span>**: string is translated.
-* **<span style="color: #fed271;">yellow</span>**: string is fuzzy.
+* **<span style="color: #c0ff00;">light-green</span>**: string is pretranslated.
 * **<span style="color: #ffa10f;">orange</span>**: string has warnings.
 * **<span style="color: #f36;">red</span>**: string has errors.
 * **<span style="color: #4d5967;">gray</span>**: string is missing.
 
-At the top of the sidebar, the translator can access Pontoon’s [search and string filters](search_filters.md).
+At the top of the sidebar, the user can access Pontoon’s [search and string filters](search_filters.md).
 
-Note that the sidebar toggles between the strings and the editing space when using the in-context editor.
+When a string is selected in the sidebar, a small icon with 4 arrows is displayed near the checkbox: this can be used to show strings that surround the selected string in the [resource](glossary.md#resource), bypassing the current filter. This is often helpful to provide more context for the localization, especially when translating missing strings.
+
+![Button to show sibling strings in sidebar](../../assets/images/pontoon/ui/sidebar_expand.png "Screenshot of the button to show sibling strings in sidebar")
 
 ### Translation workspace
 
-The translation workspace is where strings are translated.
+The translation workspace is where the translation takes place.
+
+![Translation workspace](../../assets/images/pontoon/ui/workspace_standard.png "Screenshot of the standard editor in the translation workspace")
 
-![Translation workspace](../../assets/images/pontoon/ui/workspace_standard.png)
+It’s possible to navigate sequentially through the strings by clicking the `PREVIOUS` or `NEXT` buttons located at the top of the workspace, or by using keyboard shortcuts (`ALT` + arrow down or up). The `COPY LINK` button copies in the clipboard a link to the current string.
 
-A translator can navigate sequentially by clicking the `PREVIOUS` or `NEXT` buttons located at the top of the workspace or by using keyboard shortcuts (`ALT` + arrow down or up).
+Above the editor, the user can view the source string, any comments regarding the string, and the resource path where the string is located.
```

Comment:
"any comments"... should we say "comments present in the resource" to avoid confusion with the COMMENTS section mentioned below?